content/en/includes/plugins/github/accounts-config-example.md

```diff
@@ -0,0 +1,29 @@
+In this example, you have `company-public` and `company-private` GitHub organizations, so you created two GitHub Apps, one for each org.
+* You installed the GitHub App for the `company-private` org with access to all repos in the org.
+* You installed the GitHub App for the `company-public` org with access to two specific repos in the org.
+
+Your `accounts` config looks like this:
+
+```yaml
+github:
+  plugin:
+    accounts:
+      - name: FirstAppRepo
+        organization:  company-public
+        repository: first-app-repo
+        defaultBranch: master
+        githubAppId: 9753
+        githubAppPrivateKey: encrypted:k8s!n:spin-secrets!k:github-company-public-token
```

Comment:
Instead of referring to a "token" it will be better if refer to githubApp-privatekey. For example
replcace `githubAppPrivateKey: encrypted:k8s!n:spin-secrets!k:github-company-public-token` with `githubAppPrivateKey: encrypted:k8s!n:spin-secrets!k:github-app-9753-privatekey`